core/adapters/mqtt/client.go

```diff
@@ -85,7 +85,7 @@ func monitorClient(id string, netAddr string, chcmd <-chan interface{}, ctx log.
 			cli = cmd.options
 			cmd.cherr <- nil
 		default:
-			ctx.WithField("cmd", cmd).Warn("Received unreckognized command")
+			ctx.WithField("cmd", cmd).Warn("Received unrecognized command")
```

Comment:
Maybe we should do a search/replace for this. @KtorZ misspelled this more than once.